nnpdf_data/nnpdf_data/commondata/dataset_names.yml

```diff
@@ -213,62 +213,92 @@ CHORUSNB:
   dataset: CHORUS_CC_NOTFIXED_PB_NB-SIGMARED
   variant: legacy
 CHORUSNBPb_dw_ite:
-  dataset: CHORUS_CC_NOTFIXED_PB_DW_NB-SIGMARED
-  variant: legacy
+  dataset: CHORUS_CC_NOTFIXED_PB_NB-SIGMARED
+  variant: legacy_dw
+CHORUS_CC_NOTFIXED_PB_DW_NB-SIGMARED:
```

Comment:
This trick is not going to be enough here...

Reply:
why?

Reply:
No ok it works! I'll try to fix the test asap and then we should be good

Reply:
There were indeed som additional minimal modifications needed: https://github.com/NNPDF/nnpdf/pull/2176/commits/6750cf986324500c76dd45d82e3f3367f10b24f8

Reply:
yes the second one was needed to read new format filters of datasets (with `_DW_`) which are now not existing (ie are legacy).

Reply:
Ok, so it would've been needed sooner or later. Could you add a comment about that (I think eventually the "old path" might be removed but what you added I think should remain)